charts/fluentd/values.yaml

```diff
@@ -50,6 +53,47 @@ resources:
 
 nodeSelector: {}
 
+output:
+  # Preset output type to use -
+  # Use it from here: https://github.com/fluent/fluentd-kubernetes-daemonset - For example for elasticsearch7:
+  # https://github.com/fluent/fluentd-kubernetes-daemonset/tree/master/docker-image/v1.10/debian-elasticsearch7
+  # Configuration to be passed via Environment variables
+  type: elasticsearch
+  # Chart supports custom configuration using custom configuration.
+  # Default configuration fromn https://github.com/fluent/fluentd-kubernetes-daemonset will be reset to the custom configs.
+  # Please add all neccessary configs into extraConfigs section.
+  # type: custom
+  plugins:
+    enabled: false
+    pluginsList: []
+  #   - fluent-plugin-cloudwatch-logs
+
+  # Additional Configs support
+  # extraConfigs:
```

Comment:
Exacly like this, but with a configs session, for example.

We can add by default the fluentd.conf witht the tail parameters and null on the fluentd.**, the kubernetes.conf with the metadata extractor, prometheus.conf, systemd.conf, output.conf ... etc,etc